modules/Microsoft.Synapse/workspaces/.test/managedvnet/deploy.test.bicep

```diff
@@ -0,0 +1,45 @@
+targetScope = 'subscription'
+
+// ========== //
+// Parameters //
+// ========== //
+@description('Optional. The name of the resource group to deploy for testing purposes')
+@maxLength(90)
+param resourceGroupName string = 'ms.synapse.workspaces-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to')
+param location string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints')
+param serviceShort string = 'swmin'
+
+// =========== //
+// Deployments //
+// =========== //
+
+// General resources
+// =================
+resource resourceGroup 'Microsoft.Resources/resourceGroups@2021-04-01' = {
+  name: resourceGroupName
+  location: location
+}
+
+// ============== //
+// Test Execution //
+// ============== //
+
+module testDeployment '../../deploy.bicep' = {
+  scope: resourceGroup
+  name: '${uniqueString(deployment().name)}-test-${serviceShort}'
+  params: {
+    name: '<<namePrefix>>${serviceShort}001'
+    defaultDataLakeStorageAccountName: 'adp<<namePrefix>>${serviceShort}001'
```

Comment:
I guess this dependency is missing?

Reply:
I guess we could use the `diagnosticsStorageAccount`

Reply:
This `defaultDataLakeStorageAccountName` makes me wonder if that needs to be a data lake storage.

Reply:
let me research a bit

Reply:
You're right - fortunately its just another Storage Account (configured as a datalake)

Reply:
Should I create then a storage acount with hns as dependecy?

Reply:
Jap, that would be great :)